components/pluralrules/src/rules/resolver.rs

```diff
@@ -0,0 +1,63 @@
+use super::ast;
```

Comment:
nit: note that I have not reviewed the parser and scanner in depth.  Since the spec for the language is known, and there are tests to confirm that the parser works, I am not sure that it pays to be very specific in review here.